src/components/cc-tcp-redirection/cc-tcp-redirection.types.d.ts

```diff
@@ -26,3 +26,9 @@ export interface DeleteTcpRedirection {
   namespace: string;
   sourcePort: number;
 }
+
+export interface SkeletonRedirection {
```

Comment:
nitpick: this type is not really needed. We do not need to hard type the tcp redirection used for skeleton. The `TcpRedirection` is enough.